packages/wp-story-editor/src/components/checklist/accessibility/videoOptimizationCheckbox/index.js

```diff
@@ -63,7 +63,7 @@ function VideoOptimizationCheckbox() {
     })
   );
 
-  const checked = currentUser?.meta?.web_stories_media_optimization;
+  const checked = currentUser?.meta?.webStoriesMediaOptimization;
```

Comment:
Post meta is one thing I'm not sure about yet.

For reading meta values (e.g. from media), we need to camelCase, but for updating we obviously need to use the snake_case format. That's slightly confusing.